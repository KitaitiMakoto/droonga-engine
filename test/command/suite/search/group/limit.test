#@include fixture/documents.jsons
{
  "type": "search",
  "dataset": "Default",
  "body": {
    "queries": {
      "documents": {
        "source": "Sections",
        "groupBy": "title",
        "output": {
          "elements": [
            "count",
            "records"
          ],
          "limit": 3
        }
      }
    }
  }
}
