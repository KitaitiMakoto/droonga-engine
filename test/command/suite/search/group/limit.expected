[
  "droonga.message",
  0,
  {
    "inReplyTo": "request-id",
    "statusCode": 200,
    "type": "search.result",
    "body": {
      "documents": {
        "count": 9,
        "records": [
          [],
          [],
          []
        ]
      }
    }
  }
]
